src/core/game/game_processor/game_processor.standard.ts

```diff
@@ -2045,6 +2096,15 @@ export class StandardGameProcessor extends GameProcessor {
     return await this.iterateEachStage(identifier, event, onActualExecuted);
   }
 
+  private async onHandleLevelBeginEvent(
```

Comment:
considering this is a unique event for pve only, do we have a way to install the event handler specifically for pve mode only?

Reply:
In my option, level can use in other pve mode, if i only handle this event in pve_classic game processor, i need to override the onHandleCommingInEvent every processor i use level in future.